src/components/LoginStatus/index.js

```diff
@@ -19,6 +20,9 @@ function LoginStatus(props) {
       <button onClick={() => updateLocale("en")}>English</button>
       <button onClick={() => updateLocale("el")}>Ελληνικά</button>
       {props.loggedStatus}
+      <div>
```

Comment:
You can place this in the src/Pages/TestPage.js in order to have all the components in a common place for demonstration purposes

Reply:
i moved it out of the login status. should i add a new route path on App.js which will lead to the testpage? like http://localhost:3000/test-page